playbooks/robusta_playbooks/common_actions.py

```diff
@@ -0,0 +1,28 @@
+from robusta.api import *
+
+
+class FindingOverrides(ActionParams):
+    title: str = ""
+    description: str = ""
+    severity: str = ""
+
+    """
+    :var title: Overriding finding title.
+    :var description: Overriding finding description.
+    :var severity: Overriding finding severity. Allowed values: DEBUG, INFO, LOW, MEDIUM, HIGH
+    """
+
+
+@action
+def customise_finding(event: ExecutionBaseEvent, params: FindingOverrides):
+    """
+    Overrides the created finding attribute with the provided attributes overrides.
+
+    This action does not create a new Finding, it just override the attributes of an existing Finding.
```

Comment:
Should be `overrides` (plura) not `override` (singular)

Reply:
fixed